src/query/plan/operator.cpp

```diff
@@ -366,9 +375,11 @@ std::vector<Symbol> CreateExpand::ModifiedSymbols(const SymbolTable &table) cons
 }
 
 std::string CreateExpand::ToString() const {
-  return fmt::format("CreateExpand ({}){}[{}:{}]{}({})", input_symbol_.name(),
+  const auto *maybe_edge_type_id = std::get_if<storage::EdgeTypeId>(&edge_info_.edge_type);
+  const bool is_expansion_static = maybe_edge_type_id != nullptr;
+  return fmt::format("{} ({}){}[{}:{}]{}({})", "CreateExpand", input_symbol_.name(),
                      edge_info_.direction == query::EdgeAtom::Direction::IN ? "<-" : "-", edge_info_.symbol.name(),
-                     dba_->EdgeTypeToName(edge_info_.edge_type),
+                     is_expansion_static ? dba_->EdgeTypeToName(*maybe_edge_type_id) : "<DYNAMIC>",
```

Comment:
Why not unpack the whole expression and output something like `<DYNAMIC variable>`?

Reply:
2 options:
* if expression evaluation is not simple here, rewrite the string into `<DYNAMIC parameter|property_lookup>` (something more descriptive, but not huge)
* if expression evaluation is simple here -> add the exact name from the query